### DISCUSSION TITLE
TextInput relationship

### DISCUSSION THREAD

**Discussion by yablochnyi** [filamentphp/filament#5272] [Help]
how to use this relationship? Use TextInput instead of Select
`
Forms\Components\Select::make('user')
                                    ->label('Phone')
                                    ->relationship('user', 'phone')
                                    ->searchable()
                                    ->required(),
`

**Comment by danharrin** [comment]
https://filamentphp.com/docs/2.x/forms/layout#saving-data-to-relationships
